service_capacity_modeling/models/org/netflix/evcache.py

```diff
@@ -297,16 +298,21 @@ def reserve_memory(instance_mem_gib):
         adjusted_disk_io_needed = 1.4 * adjusted_disk_io_needed
         read_write_ratio = reads_per_sec / (reads_per_sec + writes_per_sec)
 
+    needed_disk_gib = int(requirement.disk_gib.mid)
+    density_gib = compute_density_gib(
+        instance,
+        drive,
+        1,
+        max_local_disk_gib=max_local_disk_gib,
```

Comment:
I'm uncertain on if this is correct because EV cache also adjusts the count based on the reads / write IO. If someone is more familiar with this opinions are appreciated